src/components/NotesMindMapView/NotesMindMapViewComponent.js

```diff
@@ -132,12 +157,20 @@ export class NotesMindMapViewComponent extends Component {
     visNetwork.selectNote(targetNoteId);
   }
 
-  onNoteSelect = name => {
+  //changed from onNoteSelect to onNoteChange because this method will be called on changing of notes name, no "selection" logic exists
```

Comment:
Agree, this function looks like `onNoteChange`.

Let's leave the name change and remove the comment.